Makefile

```diff
@@ -4,3 +4,28 @@ include ./static-commontooling/make/lib_static_commontooling.mk
 include ./static-commontooling/make/standalone.mk
 include ./static-commontooling/make/pythonic.mk
 include ./static-commontooling/make/docker.mk
+
+all: all
+	@echo "make tools                          - Build the docker containers for all tools"
+	@echo "make tool-gsf_probe                 - Build the docker container for the gsf_probe tool"
+	@echo "make tool-extract_gsf_essence       - Build the docker container for the extract_gsf_essence tool"
+	@echo "make tool-wrap_audio_in_gsf         - Build the docker container for the wrap_audio_in_gsf tool"
+	@echo "make tool-wrap_video_in_gsf         - Build the docker container for the wrap_video_in_gsf tool"
+	@echo "make run-cmd-<tool name>            - Output the docker command required to make use of a tool container"
+	@echo "make push-<tool name>               - Push the tool image to your chosen repository using 'DOCKER_REPO=[url]'"
+
+tools: tool-gsf_probe tool-extract_gsf_essence tool-wrap_audio_in_gsf tool-wrap_video_in_gsf
+
+tool-gsf_probe: ms_docker-build-gsf_probe
+
+tool-extract_gsf_essence: ms_docker-build-extract_gsf_essence
+
+tool-wrap_audio_in_gsf: ms_docker-build-wrap_audio_in_gsf
+
+tool-wrap_video_in_gsf: ms_docker-build-wrap_video_in_gsf
+
+run-cmd-%: ms_docker-build-%
+	@echo docker run --rm -it -v $(shell pwd):/data ${MODNAME}_$*:${BUILD_TAG} \<ARGS\>
+
+push-%: ms_docker-push-%
```

Comment:
This won't currently push the images in CI. You'll need to add the relevant targets as pre-requisites of the `push` target iirc.